### ISSUE TITLE
NameError: uninitialized constant Airbrake::Rails

### ISSUE THREAD

**Issue by take-five** [airbrake/airbrake#909]
Hi,

I'm getting this error when trying to use Airbrake 8.1.4 in a project with ActiveRecord and Rack, but without Rails:

```
NameError: uninitialized constant Airbrake::Rails
  /Users/amikhailov/.rvm/rubies/ruby-2.5.3/lib/ruby/gems/2.5.0/gems/airbrake-8.1.4/lib/airbrake/rack/middleware.rb:136:in `install_active_record_hooks'
  /Users/amikhailov/.rvm/rubies/ruby-2.5.3/lib/ruby/gems/2.5.0/gems/airbrake-8.1.4/lib/airbrake/rack/middleware.rb:47:in `initialize'
```

**Comment by e-quilibre** [comment]
We have the same problem, it seems to be located in ```/lib/airbrake/rack/middleware.rb``` on line 117

```
ActiveSupport::Notifications.subscribe(
  'sql.active_record',
  Airbrake::Rails::ActiveRecordSubscriber.new
)
```

Assumption made relates active_record to rails

**Comment by kyrylo** [comment]
Should be fixed in v8.2.0.

**Comment by take-five** [comment]
Thanks a lot!
